appveyor.yml

```diff
@@ -52,7 +52,7 @@ matrix:
     - OTP_VERSION: 21.2
 
 init:
-  - ps: Update-AppveyorBuild -Version "$env:appveyor_build_version-$($env:appveyor_repo_commit.substring(0,7))"
+  - ps: Update-AppveyorBuild -Version "$env:appveyor_build_number-$env:appveyor_repo_branch-$($env:appveyor_repo_commit.substring(0,7))"
```

Comment:
It looks like intention of this line is replacing completely `version: "{build}-{branch}"`. Comment in that line that it will be refined in init would help reader.